dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -788,3 +788,7 @@ def details(self):
         print("\nModel Label Mapping:")
         for key, value in self.label_mapping.items():
             print("{}: {}".format(key, value))
+
+    def close(self):
+        tf.keras.backend.clear_session()
+        print('closing model =====================')
```

Comment:
why is closing necessary? Did we have to do it before?

Also, we should remove the print

Reply:
The reason I added it is to make sure we can close session after done with the model. Whether to keep this, I'm open for the discussion.

Reply:
IMO Definitely remove the "print"

I do think you have to close the session. A tad concerned that closing will make it so it can't be reran

Reply:
Why do we need to close the session? I thought TF2 was supposed to correct the need for session handling.

IIRC, using `tf.keras.backend.clear_session()` is like using a TF1 technique in `TF2`. I maybe wrong about this.